content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -283,23 +343,37 @@ the Pods' relative priority to determine the eviction order, because inodes and
 requests.
 
 The kubelet sorts pods differently based on whether the node has a dedicated
-`imagefs` filesystem:
+`imagefs` or `containerfs` filesystem:
 
-#### With `imagefs`
+#### With `imagefs` (`nodefs` and `imagefs` filesystems are separate)
 
-If `nodefs` is triggering evictions, the kubelet sorts pods based on `nodefs`
-usage (`local volumes + logs of all containers`).
+- If `nodefs` triggers evictions, the kubelet sorts pods based on `nodefs`
+  usage (`local volumes + logs of all containers`).
 
-If `imagefs` is triggering evictions, the kubelet sorts pods based on the
-writable layer usage of all containers.
+- If `imagefs` triggers evictions, the kubelet sorts pods based on the
+  writable layer usage of all containers.
 
-#### Without `imagefs`
+#### Without `imagefs` (`nodefs` and `imagefs` use the same filesystem)
+
+- If `nodefs` triggers evictions, the kubelet sorts pods based on their
+  total disk usage (`local volumes + logs and a writable layer of all containers`).
+
+#### With `containerfs` (`imagefs` and `containerfs` have been split)
+
+- If `containerfs` triggers evictions, the kubelet sorts pods based on
+  `containerfs` usage (`local volumes + logs and a writable layer of all containers`).
 
-If `nodefs` is triggering evictions, the kubelet sorts pods based on their total
-disk usage (`local volumes + logs & writable layer of all containers`)
+- If `imagefs` triggers evictions, the kubelet sorts pods based on the
+  `storage of images` rank, which represents the disk usage of a given image.
 
 ### Minimum eviction reclaim
 
+{{<note>}}
+You currently cannot set a custom value for the `containerfs.available` metric.
```

Comment:
```suggestion
You cannot set a custom value for the `containerfs.available` metric.
```
(I don't think “currently” helps; you could explain that “In Kubernetes x.yy&hellip;” instead, if we have an ambition to remove this limitation).

Reply:
Done.